### DISCUSSION TITLE
How to Log model events By Anonymous?

### DISCUSSION THREAD

**Discussion by kalimulhaq** [spatie/laravel-activitylog#991] [Q&A]
Is there any way to allow logging model events By Anonymous? 

Usecase: Artisan Command, Webhooks

**Comment by Gummibeer** [comment]
https://spatie.be/docs/laravel-activitylog/v4/basic-usage/logging-activity#setting-a-causer
https://spatie.be/docs/laravel-activitylog/v4/advanced-usage/define-causer-for-runtime
